MapboxVision/Services/Recording/VideoBuffer.swift

```diff
@@ -22,9 +22,13 @@ final class VideoBuffer {
     
     var chunkLimit: Int
     
-    private let settings: VideoSettings
-    private let recorder: VideoRecorder
+    var settings: VideoSettings {
```

Comment:
Same thing as with `VideoRecorder`: consider injecting settings in `startRecording`.